scripts/rsync.Dockerfile

```diff
@@ -0,0 +1,34 @@
+FROM ubuntu:20.04
```

Comment:
How large is this image? Can we start from something smaller like `alpine` or `debian` slim? 

Also, I think this should go in a `dockerfiles` folder or something like that.

Reply:
119M with debian:11-slim. Used to be 203-236M with ubuntu.